src/Lykke.Blockchains.Integrations.Contract/TransactionsExecutor/TransactionBroadcastingError.cs

```diff
@@ -1,29 +1,18 @@
 ﻿using System.Runtime.Serialization;
 using JetBrains.Annotations;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
 
 namespace Lykke.Blockchains.Integrations.Contract.TransactionsExecutor
 {
     /// <summary>
-    /// Enum describing reason of the transaction execution failure.
+    /// Enum describing reason of the transaction broadcasting failure.
     /// </summary>
     [PublicAPI]
-    public enum TransactionExecutionError
+    [JsonConverter(typeof(StringEnumConverter), typeof(CamelCaseNamingStrategy), new object[0], false)]
+    public enum TransactionBroadcastingError
```

Comment:
What is the difference between TransientFailure and Retry later? Also, may be will add something like NonTransientFailure for potential edge cases in integrations?

Reply:
TransientFailure is an execution error and will be logged as an error. RetryLater is kinda permissible and well know result and likely will be logged not as an error and different time interval will be used to retry the command. RetryLater was add to support case with Rootstock, when node just can't serve more operations at the moment.